browse/templates/user_banner.html

```diff
@@ -44,6 +44,7 @@ <h1>Open Access Giving Week</h1>
 
 {%- macro script(request_datetime) -%}
   <link rel="stylesheet" type="text/css" media="screen" href="{{ url_for('static', filename='css/slider.css') }}?v=1.1" />
+  <script type="text/javascript" src="{ url_for('static', filename='js/donate.js') }}"></script>
```

Comment:
This isn't working. It probably needs to be:

`src="{{ url_for('static', filename='js/donate.js') }}`